docs/index.yaml

```diff
@@ -1,6 +1,33 @@
 apiVersion: v1
 entries:
   hpe-array-exporter:
+  - annotations:
+      artifacthub.io/license: Apache-2.0
+      artifacthub.io/links: |
+        - name: Documentation
+          url: https://hpe-storage.github.io/array-exporter
+      artifacthub.io/prerelease: "false"
+    apiVersion: v2
+    appVersion: 1.0.0
+    created: "2022-01-25T11:10:10.630795-08:00"
+    description: A Helm chart for installing the HPE Storage Array Exporter for Prometheus
+    digest: b16ff635abdab9e55c2e0fa51c54699145dfebd0f31b8ad53a4c98c881be7cc0
+    home: https://hpe.com/storage/containers
+    icon: https://raw.githubusercontent.com/hpe-storage/co-deployments/master/docs/assets/hpedev.png
+    keywords:
+    - HPE
+    - Storage
+    - Prometheus
+    kubeVersion: '>= 1.18.0'
```

Comment:
check with @datamattsson if we still need the kubeVersion semVer compare as it might cause issues on several downstream distros